packages/mux-uploader/README.md

```diff
@@ -55,28 +55,178 @@ If you are using ECMAScript modules, you can also load the `mux-uploader.mjs` fi
 <script type="module" src="https://unpkg.com/@mux/mux-uploader/dist/mux-uploader.mjs"></script>
 ```
 
-## Usage
+# Usage
+
+In these examples `"authenticated-url"` is the `url` property in the response returned from [creating a Direct Upload](https://docs.mux.com/api-reference/video#operation/create-direct-upload). Creating a direct upload happens server-side directly to the Mux API.
+
+The `url` for a Direct Upload looks like `"https://storage.googleapis.com/video..."`.
```

Comment:
I would make it more clear that "authenticated-url" is their own url with something like

`https://my-authenticated-url.com/storage?long-ass-url-params`

bad habit but I skim a lot and this would look like it's a real thing `endpoint="authenticated-url"` as attribute if skipping the 

> In these examples `"authenticated-url"` is the `url` property in

Reply:
I'll clean this up. The backticks also make me think it's either a variable name or the name of the property itself but we're actually instead referring to the value.